recipes/tagine.md

```diff
@@ -0,0 +1,58 @@
+# Lubes' Vegetable Tagine
```

Comment:
Then ill be adding my name to the Fatoush.
and it will be " The Amaizng Mario's Fatoush " :sunglasses: